rust/actyx/certs/src/bin/ax-cert.rs

```diff
@@ -107,9 +193,111 @@ fn create_dev_cert(opts: DevCertOpts) -> anyhow::Result<()> {
 }
 
 fn create_app_license(opts: AppLicenseOpts) -> anyhow::Result<()> {
-    let license = SignedAppLicense::new(opts.actyx_private_key, opts.email, opts.app_id, opts.expires_at, None)?;
+    let expiration_date = opts
+        .expires_at
+        .or(opts.expires_in)
+        .unwrap_or_else(|| Utc::now() + chrono::Duration::days(1));
```

Comment:
either document the default of one day or remove it and raise an error if both args are absent

Reply:
Documented the precedence of `--expires-at` and removed the default. Thanks for the suggestion!